src/sidebar/store/modules/test/annotations-test.js

```diff
@@ -236,8 +415,88 @@ describe('sidebar/store/modules/annotations', function() {
       );
     });
 
+    it('should set basic default properties on a new/empty annotation', () => {
```

Comment:
This test and one below have an unpleasant number of assertions, and I typically try to keep that more concise by writing separate tests or unrolling with `forEach`/testcases. Neither of these tests seemed immediately amenable to these conventions; I rewrote them a few times and found that they got more unwieldy, not less.

Reply:
You are matching a bunch of different properties on one object. You can do this in one assertion using `assert.match`. I think the approach taken here of writing several closely related asserts is absolutely fine though.